src/reynir_correct/errtokenizer.py

```diff
@@ -2614,10 +2614,10 @@ def late_fix_merges(
     # Where the error is placed by default on the first word instead of
     # the word that changes (also could be many changes!)
     for token in token_stream:
-        # Add annotations if annotation has disappeared
+        # Add annotations if token text has been changed, excluding soft hyphen deletion
         if (
             token.original
-            and token.txt.strip() != token.original.strip()
+            and token.txt.strip() != token.original.strip().replace("\u00AD", "")
```

Comment:
Maybe a more general way to do this would be to import the unicode_replacement() function from tokenizer and use that on the original token string? That would catch all Unicode-level replacements we do, for instance joining separate accent marks with vowels.

Reply:
Done!